elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -9,86 +9,96 @@ $pfe-cta--Color--fallback: #003366;
   // regular variables
   --pfe-cta--BorderRadius: 0;
   --pfe-cta__arrow--Display:  inline;
-  --pfe-cta__arrow--Padding:  0 .125rem 0 .375rem;
+  --pfe-cta__arrow--Padding:  8px;
   --pfe-cta--BackgroundColor:         transparent;
   --pfe-cta--BackgroundColor--focus:  transparent;
   --pfe-cta--SecondaryBackgroundColor--focus: rgba(21,21,21, .05);
   --pfe-cta--BorderColor:             transparent;
   --pfe-cta__inner--BorderColor:      transparent;
-  --pfe-cta--Color:   pfe-color(ui-link);
+  --pfe-cta--Color:   #{pfe-color(ui-link)};
   --pfe-cta--Padding: .6rem 0;
 
+
+
   // hover variables
   --pfe-cta--BackgroundColor--hover:  transparent;
   --pfe-cta--BorderColor--hover: transparent;
-  --pfe-cta--Color--hover:  pfe-color(ui-link--hover);
+  --pfe-cta--Color--hover:  #{pfe-color(ui-link--hover)};
 
   // focus variables
   --pfe-cta--BorderColor--focus: transparent;
   --pfe-cta__inner--BorderColor--focus: transparent;
-  --pfe-cta--Color--focus:  pfe-color(ui-link--focus);
+  --pfe-cta--Color--focus:  #{pfe-color(ui-link--focus)};
 
   // properties
   display: inline-block;
-  font-family: pfe-var(font-family--heading);
-  font-size:   pfe-var(font-size);
-  font-weight: pfe-var(font-weight--bold);
-  line-height: pfe-var(line-height);
-  background-color: pfe-local(BackgroundColor);
-  border-color:     pfe-local(BorderColor);
+  font-family: #{pfe-var(font-family--heading)};
+  font-size:   #{pfe-var(font-size)};
+  font-weight: #{pfe-var(font-weight--bold)};
+  line-height: #{pfe-var(line-height)};
+  background-color: #{pfe-local(BackgroundColor)};
+  border-color:     #{pfe-local(BorderColor)};
   position: relative;
-  transition: .3s ease padding; // padding pfe-var(animation-timing);
+  transition: .3s ease padding; // padding #{pfe-var(animation-timing)};
   line-height: inherit;
   vertical-align:  middle;
-  background-color: pfe-local(BackgroundColor);
-  border-radius:    pfe-local(BorderRadius);
-  border:           pfe-var(ui--border-width) pfe-var(ui--border-style) pfe-local(BorderColor);
+  background-color: #{pfe-local(BackgroundColor)};
+  border-radius:    #{pfe-local(BorderRadius)};
+  border:           #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor)};
 
+  //  --pfe-cta__arrow--Padding:  0 .125rem 0 .375rem;
+  //calc( #{pfe-local($cssvar: Padding, $region: arrow)}
   .pfe-cta--arrow {
-    display: pfe-local($cssvar: arrow--Display, $region: arrow);
-    padding: pfe-local($cssvar: Padding, $region: arrow);
+    display: #{pfe-local($cssvar: arrow--Display, $region: arrow)};
+    padding: 0;
+    padding-right: calc( 13px + #{var(--pfe-cta__arrow--Padding)} );
+    margin-right:  calc( -13px - #{var(--pfe-cta__arrow--Padding)} );
```

Comment:
this part is the WIP, i can back it out. Was just fussing with negative margins to see if we could solve the wrapping arrow this way